app/src/main/java/com/duckduckgo/app/survey/ui/SurveyActivity.kt

```diff
@@ -99,6 +101,7 @@ class SurveyActivity : DuckDuckGoActivity() {
     private fun loadSurvey(url: String) {
         binding.progress.show()
         webView.loadUrl(url)
+        Timber.d("Survey URL: $url")
```

Comment:
should this be removed?

Reply:
yes, I will remove it as I added it for debug purposes and forgot to remove it.